### PR TITLE
Update opensource.md

### PR DESCRIPTION
**Pull request by AlexGhiondea**
Add updates on how the code owners file supports auto-label on PRs.

### CODE REVIEW

File: docs/policies/opensource.md [hunk 1]
```diff
@@ -73,7 +74,7 @@ Use the following rules to ensure that we can use CODEOWNERS for both GitHub and
 # Service teams
 /sdk/azconfig/   @person3 @person4
 /sdk/keyvault/   @person5 @person6
-/sdk/servicebus/ @person7 @person8
+/sdk/servicebus/ @person7 @person8 %label
```
Comment (weshaggard) [inline docs/policies/opensource.md:77]: Lets wait until we figure out the working pattern :)

Comment (AlexGhiondea) [reply docs/policies/opensource.md:77]: Yeah 🙂.

Comment (joshfree) [reply docs/policies/opensource.md:77]: presumably we have a working pattern now and we can move forward with this PR?

Comment (AlexGhiondea) [reply docs/policies/opensource.md:77]: Yes. I will push an update.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**